services/QuillLMS/client/app/bundles/Teacher/components/progress_reports/student_overview.jsx

```diff
@@ -116,7 +116,7 @@ export default class extends React.Component {
         newRow['Average Score'] = countAndAverage.average;
         csvReportData.push(newRow);
       });
-      downloadReportOrLoadingIndicator = <CSVDownloadForProgressReport data={csvReportData} preserveCasing={'t'} />;
+      downloadReportOrLoadingIndicator = <CSVDownloadForProgressReport data={csvReportData} preserveCasing={true} />;
```

Comment:
We might want to double check that this works - I think it should but this isn't something that's coming out of the query.

Reply:
I did a spot check of two files generated one with 't' and one with `true`